src/components/piece/basicPieces.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import Piece from './piece';
+
+ class Pieces extends React.Component {
```

Comment:
i'm not sure if there's a future plan here that would require this to be a class component, but it seems this could very easily be a functional component.it would be a very short file to start with.